configs/rollup/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@slash/rollup-config",
```

Comment:
I think in order for us to make this package public, we have to change the package name. (e.g. `@toss/rollup-config`)